include/IFrameBuffer.h

```diff
@@ -50,12 +53,31 @@ class IRenderBuffer;
 class IFrameBuffer : public virtual IReferenceCounted
 {
     public:
+		//! Attaches given texture to given attachment point.
+		/** @param attachmentPoint Identifies attchment point.
+		@param tex Texture being attached.
+		@param mipMapLayer
+		@param layer
+		@returns Whether attachment operation went succesfully.
```

Comment:
Only returns if the attachment has been attached (glFramebuffer* called) and the FBO is queued up for rebinding.
A return value of TRUE does not mean that the attachment color format is renderable or that the combination of attachments is valid.
Only after rebindRevalidate() is called by the driver internally or by the user manually do the attachments drawn into by the FrameBuffer change.